src/main/java/com/github/maxopoly/WurstCivTools/effect/LeafShears.java

```diff
@@ -0,0 +1,118 @@
+/**
+ * @author Aleksey Terzi
+ *
+ */
+
+package com.github.maxopoly.WurstCivTools.effect;
+
+import org.bukkit.Bukkit;
+import org.bukkit.Location;
+import org.bukkit.Material;
+import org.bukkit.World;
+import org.bukkit.block.Block;
+import org.bukkit.entity.Player;
+import org.bukkit.event.block.BlockBreakEvent;
+import org.bukkit.inventory.ItemStack;
+
+import vg.civcraft.mc.citadel.Citadel;
+import vg.civcraft.mc.citadel.PlayerState;
+import vg.civcraft.mc.citadel.reinforcement.PlayerReinforcement;
+import vg.civcraft.mc.citadel.reinforcement.Reinforcement;
+
+import com.github.maxopoly.WurstCivTools.WurstCivTools;
+
+public class LeafShears extends WurstEffect {
+	private int clearCubeSize;
+
+	public LeafShears(int clearCubeSize) {
+		super();
+		
+		this.clearCubeSize = clearCubeSize;
+	}
+
+	@Override
+	public void handleBreak(Player p, BlockBreakEvent e) {
+		if(e.isCancelled()) {
+			return;
+		}
+		
+		final Player player = p;
+		
+		if(player == null) {
+			return;
+		}
+		
+		if(!isLeaf(e.getBlock())) {
+			return;
+		}
+		
+		e.setCancelled(true);
+		e.getBlock().setType(Material.AIR);
```

Comment:
> Pretty sure this needs to be wrapped in the Citadel check; abandon the whole attempt if this block is citadel protected on an inaccessible group.  
    
> Also we dont want to cancel the event if the player doesnt have bypass, so it still applies normal reinforcement damage

This is already working correctly.  
When Citadel is preventing block breaking because of another group or /ctb disabled - it is cancelling event.

But since WurstCivTools's BlockBreak event has HIGH priority and since WurstCivTools has softdepend on Citadel - then if Citadel is cancelling this event - it is even not firing in WurstCivTools.  
    
You can test this.